src/lib/stitching/exchange/v2/stitching.ts

```diff
@@ -146,6 +146,60 @@ export const exchangeStitchingEnvironment = ({
         return Boolean(impulseConversationId)
       },
     },
+    conversation: {
+      fragment: gql`
+        fragment CommerceOrderConversation on CommerceOfferOrder {
+          impulseConversationId
+        }
+      `,
+      resolve: async (order, _args, context, info) => {
+        const { impulseConversationId } = order
+        if (!impulseConversationId) return null
+
+        return info.mergeInfo.delegateToSchema({
+          schema: localSchema,
+          operation: "query",
+          fieldName: "conversation",
+          args: { id: impulseConversationId },
+          context,
+          info,
+          transforms: [],
+          //   // Wrap document takes a subtree as an AST node
+          //   new WrapQuery(
+          //     // path at which to apply wrapping and extracting
+          //     ["conversation"],
+          //     (subtree: SelectionSetNode) => ({
+          //       // we create a wrapping AST Field
+          //       kind: Kind.FIELD,
+          //       name: {
+          //         kind: Kind.NAME,
+          //         value: "conversation",
+          //       },
+          //       arguments: [
+          //         {
+          //           kind: Kind.ARGUMENT,
+          //           name: {
+          //             kind: Kind.NAME,
+          //             value: "id",
+          //           },
+          //           value: {
+          //             kind: Kind.STRING,
+          //             value: impulseConversationId,
+          //           },
+          //         },
+          //       ],
+          //       // Inside the field selection
+          //       selectionSet: subtree,
+          //     }),
+          //     // how to process the data result at path
+          //     (result) => {
+          //       return result.conversation
+          //     }
+          //   ),
+          // ],
+        })
+      },
+    },
```

Comment:
Any reason this is commented out?

Reply:
yes, it's left over from a draft PR which used a previous reverted and unreverted commit. this is allowing is to pass `order.conversation` to `me.conversation` but now we are just delegating to the new root conversation field.